package.json

```diff
@@ -29,50 +32,45 @@
     "prepublish": "yarn run build"
   },
   "dependencies": {
-    "babel-plugin-react-flow-props-to-prop-types": "0.15.0",
-    "babel-runtime": "6.26.0",
-    "invariant": "2.2.2",
-    "memoize-one": "3.0.1",
-    "prop-types": "15.5.10",
-    "prop-types-extra": "1.0.1",
-    "raf-schd": "2.0.2",
-    "react-motion": "0.5.0",
-    "react-redux": "5.0.6",
-    "redux": "3.7.2",
-    "redux-thunk": "2.2.0",
-    "reselect": "3.0.1"
+    "babel-runtime": "^6.26.0",
+    "invariant": "^2.2.2",
+    "memoize-one": "^3.0.1",
+    "raf-schd": "^2.0.2",
+    "react-motion": "^0.5.2",
+    "react-redux": "^5.0.6",
+    "redux": "^3.7.2",
+    "redux-thunk": "^2.2.0",
+    "reselect": "^3.0.1"
   },
   "devDependencies": {
-    "@atlaskit/css-reset": "1.1.5",
-    "@storybook/react": "3.2.10",
-    "babel-cli": "6.26.0",
-    "babel-eslint": "8.0.0",
-    "babel-plugin-transform-class-properties": "6.24.1",
-    "babel-plugin-transform-export-extensions": "6.22.0",
-    "babel-plugin-transform-object-rest-spread": "6.26.0",
-    "babel-plugin-transform-runtime": "6.23.0",
-    "babel-preset-es2015": "6.24.1",
-    "babel-preset-flow": "6.23.0",
-    "babel-preset-react": "6.24.1",
-    "enzyme": "2.9.1",
-    "eslint": "4.7.2",
-    "eslint-config-airbnb": "15.1.0",
-    "eslint-plugin-import": "2.7.0",
-    "eslint-plugin-jest": "21.1.0",
-    "eslint-plugin-jsx-a11y": "5",
-    "eslint-plugin-react": "7.3.0",
-    "flow-bin": "0.52.0",
-    "flow-copy-source": "1.2.1",
-    "jest": "21.1.0",
-    "raf-stub": "2.0.1",
-    "react": "15.6.1",
-    "react-dom": "15.6.1",
-    "react-test-renderer": "15.6.1",
-    "rimraf": "2.6.2",
-    "styled-components": "2.1.2"
-  },
-  "peerDependencies": {
-    "react": ">=15.4.0"
```

Comment:
keep react as a peerdep rather than a devdep?

Reply:
Kept it as both - i think we needed it for storybook